modules/installing-aws-load-balancer-operator.adoc

```diff
@@ -19,15 +17,14 @@ You can install the AWS Load Balancer Operator from the OperatorHub by using the
 . Navigate to *Operators* → *OperatorHub* in the {product-title} web console.
 . Select the *AWS Load Balancer Operator*. You can use the *Filter by keyword* text box or use the filter list to search for the AWS Load Balancer Operator from the list of Operators.
 . Select the `aws-load-balancer-operator` namespace.
-. Follow the instructions to prepare the Operator for installation.
-. On the *AWS Load Balancer Operator* page, click *Install*.
+. (Optional) Follow the instructions to prepare the Operator for installation in custom scenarios when you want to use a security token service (STS) or a User Provisioned Infrastructure (UPI).
```

Comment:
Per IBM Style, use "Optional: " to indicate an optional step (see https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#indicating-optional-and-conditional-steps)

We also don't use the abbreviations UPI and IPI in the docs and do hyphenated lowercase (see https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#user-provisioned-infrastructure)

```suggestion
. Optional: Follow the instructions to prepare the Operator for installation in custom scenarios when you want to use a security token service (STS) or user-provisioned infrastructure.
```